src/main/java/net/minecraftforge/network/NetworkConstants.java

```diff
@@ -20,7 +20,7 @@
 public class NetworkConstants
 {
     public static final String FMLNETMARKER = "FML";
-    public static final int FMLNETVERSION = 2;
+    public static final int FMLNETVERSION = 3;
```

Comment:
It may be good to add a comment here explaining what necessitated the bump to version 3, for ease of future readers.

Reply:
Added in next push